AwsEncryptionSDK/runtimes/net/Generated/AwsEncryptionSdk/CollectionOfErrors.cs

```diff
@@ -1,16 +1,27 @@
 // Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 // Do not modify this file. This file is machine generated, and any changes to it will be overwritten.
+
 using System;
-using AWS.Cryptography.EncryptionSDK;
+using System.Collections.Generic;
+
 namespace AWS.Cryptography.EncryptionSDK
 {
     public class CollectionOfErrors : Exception
     {
         public readonly System.Collections.Generic.List<Exception> list;
-        public CollectionOfErrors(System.Collections.Generic.List<Exception> list, string message) : base(message) { this.list = list; }
+        public CollectionOfErrors(System.Collections.Generic.List<Exception> list, string message) : base(message + $"\n List: \n{ListAsString(list)}") { this.list = list; }
         public CollectionOfErrors(string message) : base(message) { this.list = new System.Collections.Generic.List<Exception>(); }
         public CollectionOfErrors() : base("CollectionOfErrors") { this.list = new System.Collections.Generic.List<Exception>(); }
+
+        private static string ListAsString(List<Exception> list)
```

Comment:
Non blocking suggestion to maintain consistency in this class:
```suggestion
        private static string ListAsString(System.Collections.Generic.List<Exception> list)
```
And remove the import

Reply:
I am OK with this style divergence.
This is Beneration.
Ideally, we would go back and refactor Smithy-Dafny such that it always generates this.
I am not sure if/when we will do that,
as it may happen as part of other Smithy-Dafny feature development.

But this logic was written by a human;
it was not machine generated.

It is acceptable for it to appear as such;
it may even be preferred, 
as it helps clarify what was benerated from what was generated.